src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -10,6 +10,7 @@
     <RootNamespace>System.Collections.Tests</RootNamespace>
     <DefineConstants Condition="'$(TargetGroup)'=='netstandard1.7'">$(DefineConstants);netstandard17</DefineConstants>
```

Comment:
Something along the lines of:
```
    <Project Include="System.Collections.Tests.csproj">
      <TargetGroup>netcoreapp1.1</TargetGroup>
      <TestTFMs>netcoreapp1.1</TestTFMs>
    </Project>
```

Then to build and run the tests you either needs to `msbuild System.Collections.Tests.csproj /p:TargetGroup=netcoreapp1.1`